pages/common/flac.md

```diff
@@ -0,0 +1,16 @@
+# flac
+
+> Encodes, decodes and tests flac files.
+
+
+- Encode a wav file to flac, writing to /path/to/file.flac:
+
+`flac {{/path/to/file.wav}}`
+
+- Decode a flac file to wav, specifying the output file:
+
+`flac -d --output-name {{/path/to/output.wav}} {{/path/to/input.flac}}`
+
+- Test a flac file:
```

Comment:
What does it test for ?

Reply:
for correct encoding - if there are any remarkable errors in the file, they are reported. I added a short remark in the example.